bazel_tools/scala.bzl

```diff
@@ -42,6 +42,7 @@ version_specific = {
         "-Xlint:unsound-match",
         "-Xlint:by-name-right-associative",  # will never be by-name if used correctly
         "-Xfuture",
+        "-language:higherKinds",
```

Comment:
The alternative is to silence unused imports on 2.13 either selectively or completely. This seemed nicer but open to changing it.